docs/pages/versions/v42.0.0/sdk/facebook.md

```diff
@@ -174,34 +85,14 @@ async function logIn() {
 }
 ```
 
-Given a valid Facebook application ID in place of `<APP_ID>`, the code above will prompt the user to log into Facebook then display the user's name. This uses React Native's [fetch](https://reactnative.dev/docs/network.html#fetch) to query Facebook's [Graph API](https://developers.facebook.com/docs/graph-api).
-
-### `Facebook.logOutAsync()`
-
-Logs out of the currently authenticated session.
-
-### `Facebook.getAuthenticationCredentialAsync()`
-
-Returns the `FacebookAuthenticationCredential` object if a user is authenticated, and `null` if no valid authentication exists.
-
-You can use this method to check if the user should sign in or not.
-
-#### Returns
-
-- A promise that resolves a [`FacebookAuthenticationCredential`][#facebookauthenticationcredential].
-
-```tsx
-async function toggleAuthAsync() {
-  const auth = await Facebook.getAuthenticationCredentialAsync();
+## API
 
-  if (!auth) {
-    // Log in
-  } else {
-    // Log out
-  }
-}
+```js
+import * as Facebook from 'expo-facebook';
 ```
 
+<APISection packageName="expo-facebook" />
+
 ## Types
 
 ### FacebookAuthenticationCredential
```

Comment:
@cruzach It looks like `FacebookAuthenticationCredential` type is duplicated on the page. `APISection` outputs this type too:
- https://docs.expo.io/versions/latest/sdk/facebook/#facebookauthenticationcredential
- https://docs.expo.io/versions/latest/sdk/facebook/#facebookauthenticationcredential-1

Reply:
thanks @Simek ! fixed here- https://github.com/expo/expo/commit/260aff86930bcf48b6037dfcea5b988f5acfe761